### DISCUSSION TITLE
Latest Async Build with [dxvk] Implement framebuffer cache Artifacts (Package) #791

### DISCUSSION THREAD

**Discussion by ghost** [Sporif/dxvk-async#28] [Q&A] [answered]
Hi there, would someone kindly help me by build Dxvk Async with the latest commit included #791
( [dxvk] Implement framebuffer cache Artifacts (Package) #791)

@rbernon @Sporif @Kron4ek

Thank you :)

**Comment by Sporif** [comment, marked as answer]
Here you go: [dxvk-async-1.9.2_r39_g06eb5238.tar.gz](https://github.com/Sporif/dxvk-async/files/7519328/dxvk-async-1.9.2_r39_g06eb5238.tar.gz)

**Reply by ghost** [reply]
Thank you 😊

**Reply by frusio** [reply]
Thanks, Sporif!
